src/masonite/drivers/notification/MailDriver.py

```diff
@@ -0,0 +1,66 @@
+"""Mail driver Class."""
+
+from ...exceptions.exceptions import NotificationException
+from .BaseDriver import BaseDriver
+
+
+class MailDriver(BaseDriver):
```

Comment:
Maybe we need to use the `Mailable` class here to build the email notification ..

Reply:
I have used the Mailable class but not in the way you meant maybe. 
When you build the notification, the mail_to method should return a Mailable